### PR TITLE
Add dates and sponsorship for 2016

### LINKED ISSUE #171: Update to reflect 2016 dates
10-11 March 2016.

- [ ] update website
- [ ] tweet
- [x] update sponsor doc
- [ ] lanyrd (if we've done it already)

### PR DESCRIPTION
**Pull request by imsickofmaps**
Fixes #171 and #172

### CODE REVIEW

**Comment by imsickofmaps** [comment]
Ready for review

File: sponsors/index.html [hunk 1]
```diff
@@ -46,21 +39,21 @@
...
         <a href='/sponsors/hetzner.html'>
             <img src="/img/sponsors/hetzner.large.png" class="img-responsive sponsor-logo" alt="Hetzner">
         </a>
-        <b>Coffee - Wednesday &amp; Friday</b>
+        <b>Coffee</b>
     </div>
-    <div class="col-md-2 text-center">
+    <!-- <div class="col-md-2 text-center">
         <a href='/sponsors/oracle.html'>
             <img src="/img/sponsors/oracle.large.png" class="img-responsive sponsor-logo" alt="Oracle">
```
Comment (vhata) [inline sponsors/index.html]: This doesn't show for me when I run `jekyll serve` locally.

Comment (imsickofmaps) [inline sponsors/index.html]: yeah, because it's a HTML commented out section. `<!-- `

**Comment by vhata** [comment]
I meant the Hetzner logo doesn't show, sorry.  If it works for you, :+1:

**Pull request merged**